uber/scheduler.py

```diff
@@ -8,14 +8,13 @@
 # don't start the email sending in the main thread. everything else is cool.
 # NOTE: only applies to cherrypy process ONLY, doesn't apply when started via 'sep' or other processes.
 # uncomment this if you want to do email sending in its own process
-# _default_cherrypy_exclude_categories = ["automated_email_sending"]
-_default_cherrypy_exclude_categories = None
+_default_cherrypy_exclude_categories = ["automated_email_sending"]
```

Comment:
this is the magic.  main process won't start the automated_email_sending category, but we will explicitly do it in the sideboard entry point